debian/rules

```diff
@@ -32,7 +32,7 @@ CXX := $(DEB_HOST_GNU_TYPE)-g++
 # at https://www.debian.org/doc/debian-policy/ch-source.html#s-debianrules-options
 ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
     NUMJOBS = $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
-    MAKEFLAGS += -j $(NUMJOBS)
+    MAKEFLAGS += -j$(NUMJOBS)
```

Comment:
This was recently done by @ottok: 325c9ce9b39a7695bf217cd92fc36034b2e8207b

Reply:
apparently this was just re-conforming to the debian policy docs with no material impacts.